src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3472,7 +3501,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                                            const Control::FontSizeChangedArgs& args)
     {
         // scale the selection markers to be the size of a cell
-        auto scaleMarker = [args, dpiScale{ SwapChainPanel().CompositionScaleX() }](const Windows::UI::Xaml::Shapes::Path& shape) {
+        const auto dpiScale = SwapChainPanel().CompositionScaleX();
+        auto scaleMarker = [args, &dpiScale](const Windows::UI::Xaml::Shapes::Path& shape) {

```

Comment:
It's a little ironic that the `args` gets copied but `dpiScale`, which is a primitive scalar, gets referenced. 😅 If anything it should be the other way around. (To be fair, the AddRef & Release calls are very cheap, not worth worrying too much about.)